provider_gsuite.go

```diff
@@ -0,0 +1,217 @@
+package jwtauth
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/mitchellh/mapstructure"
+	"golang.org/x/oauth2/google"
+	"golang.org/x/oauth2/jwt"
+	admin "google.golang.org/api/admin/directory/v1"
+	"google.golang.org/api/option"
+)
+
+// GSuiteProvider provides G Suite-specific configuration and behavior.
+type GSuiteProvider struct {
```

Comment:
For improved readability, please leave a new line between each comment/variable pair. These are short enough for inline comments too, probably.

Reply:
Yep, that reads better. Done in ddda226fe68cf71f6c351ffe27cbef1ba09f109a.